app/services/model_controller/notifier.rb

```diff
@@ -11,43 +13,28 @@ def success_action
     def success_insert(item)
       respond_to do |format|
         format.html { redirect_to admin_edit_uri(item) }
-        format.json {
-          if params.has_key?(:create_and_return)
-            data = { :redirect_to => admin_list_uri(item.class) }
-          else
-            data = { :form_action => admin_update_uri(item) }
-          end
-          success_message(item, I18n.t('rademade_admin.success_insert_message'), data)
-        }
+        format.json { success_message(item, I18n.t('rademade_admin.success_insert_message'), success_data) }
       end
     end
 
     def success_update(item)
       respond_to do |format|
         format.html { redirect_to admin_edit_uri(item) }
-        format.json {
-          data = {}
-          data[:redirect_to] = admin_list_uri(item.class) if params.has_key?(:create_and_return)
-          success_message(item, I18n.t('rademade_admin.success_update_message'), data)
-        }
+        format.json { success_message(item, I18n.t('rademade_admin.success_update_message'), success_data) }
       end
     end
 
     def success_delete(item)
       respond_to do |format|
         format.html { redirect_to admin_list_uri(item) }
-        format.json {
-          success_message(item, I18n.t('rademade_admin.success_delete_message'))
-        }
+        format.json { success_message(item, I18n.t('rademade_admin.success_delete_message')) }
       end
     end
     
     def success_status_change(item)
       respond_to do |format|
         format.html { redirect_to admin_list_uri(item) }
-        format.json {
-          success_message(item, t('rademade_admin.success_status_update_message'))
-        }
+        format.json { success_message(item, t('rademade_admin.success_status_update_message')) }
```

Comment:
Line is too long. [96/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.